ui/jobs/buff_tracker.ts

```diff
@@ -199,18 +200,24 @@ export class Buff {
     this.active.removeCallback();
     this.active = null;
   }
+
+  onCooldown(seconds: number, source: string): void {
+    this.clearCooldown(source);
+    this.addCooldown(source, seconds);
```

Comment:
FWIW, when using Battle Voice on myself, because the ability occurs before the effect, it briefly flashes 180 before 20.  Do you think it's worth changing the `addCooldown` to add like 0.5 or 1 second minimum before showing the cooldown just to avoid this race?  I also have the setting where my minimum time is set to 9999.  (Or is there some other way to avoid this?)

Reply:
Well, I'll add something to ensure `secondsUntilShow` at least 1